doc/reference/configuration/configuration_reference.rst

```diff
@@ -492,8 +492,192 @@ The ``database`` section defines database-specific configuration parameters, suc
     | Default: false
     | Environment variable: TT_DATABASE_USE_MVCC_ENGINE
 
+..  _configuration_reference_flightrec:
 
+flightrec
+---------
 
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    Configuring ``flightrec`` parameters is available in the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+The ``flightrec`` section describes options related to the :ref:`flight recorder <enterprise-flight-recorder>` configuration.
+Note that all options are dynamic and can be changed at runtime.
+
+..  NOTE::
+
+    ``flightrec`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+To enable flight recorder, set the ``flightrec.enabled`` option to ``true`.
+
+..  _configuration_reference_flightrec_enabled:
+
+..  confval:: flightrec.enabled
+
+    Whether the :ref:`flight recorder <enterprise-flight-recorder>` is enabled.
+
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_FLIGHTREC_ENABLED
+
+
+..  TODO not implemented yet
+    .. _config-directory:
+
+    ..  _configuration_reference_flightrec_dir:
+
+    ..  confval:: flightrec.dir
+
+        Specify the directory used to store flight recordings (``*.ttfr`` files).
+
+        | Type: string
+        | Default: :ref:`memtx_dir <cfg_basic-memtx_dir>`
+        | Environment variable: TT_FLIGHTREC_DIR
+
+
+..  _configuration_reference_flightrec_logs:
+
+Logs
```

Comment:
I'd drop grouping of options inside the reference. We can add some information about different types of options to the topic. For example, create a new `Configuring a flight recorder` section and make a short overview of options.